docker/upload_es_index.py

```diff
@@ -0,0 +1,19 @@
+from tl.utility.utility import Utility
```

Comment:
I think the initialization of elasticsearch should not be the responsibility of the backend's container. Either we extend the elasticsearch image to include Python and the initialization script, or we create a fourth image that just initializes elasticsearch. Having the backend container responsible for elasticsearch initialization can be *very confusing* in a few months.

Reply:
I don't think so. As elastic search is one of the necessary part of backend processing, we have to ensure the ES is fully up before we allow user to make the search. 
Also, installing table linker in another position is really time and space wasting as the part here is only needed. This problem can be resolved by copying all codes necessary to prevent installing full table linker (if needed).
But still, I think put the things together is a better option as it prevent duplicates codes from table linker and can also ensure the backend can fetech things successfully during wikification.

Reply:
It is only just few lines initialization and there is no code changes on any backend core part so I don't think it will make confusions.

Reply:
I'd much rather not mix these responsibilities. The backend needs to handle backend related stuff. One of its tasks may be to fail if elasticsearch is not up, just like it should fail if Postgres is not up after a reasonable amount of time. I don't think it should be responsible for initializing the database (the Postgres container does this on its own with an init script) or elasticsearch.

Init containers are very often used in Kubernetes. Docker compose does not support them directly (hence my suggestion to create just one container that runs elasticsearch and also initializes it). It can be achieved relatively simply, as explained here: https://github.com/docker/compose/issues/6855#issuecomment-629737199 .

Reply:
ok, i will update